docs/contributing-to-registry.md

```diff
@@ -0,0 +1,66 @@
+>**Note**: Please ignore the doc for now as it's a work in progress. We are still working on creating the Bicep public registry, and the tool mentioned in the doc has not been published yet.
+
+# Contributing to Bicep public registry
+
+Currently, we only accept contributions from Microsoft employees. The guide was created to help teams within Microsoft with the development of Bicep public registry modules. External customers can propose a new module by opening an [issue]() in the [Azure/azure-quickstart-templates](https://github.com/Azure/azure-quickstart-templates/tree/master/modules) repository.
+
+## Prerequisite
+- Create a fork of the [Azure/azure-quickstart-templates](https://github.com/Azure/azure-quickstart-templates/tree/master/modules) repository and clone the fork to your local machine.
+<!-- TODO: Add link to Nuget once the tool is published -->
+- Install the [bicep-registry-module]() .NET tool by running:
```

Comment:
My thought is that Bicep users should already have .NET installed on their machine since it's required by the language server. One benefit of shipping this as a .NET tool is that we don't have to worry about create an executable for each platform. Besides, updating the tool is as simple as running `dotnet tool update`; otherwise, users will need to manually download the executable and replace the old version in the local file system.

Reply:
I'm ok starting out as a .NET tool and seeing what feedback we get